.github/workflows/vk-gl-cts-nightly.yml

```diff
@@ -12,7 +12,6 @@ jobs:
   build:
     runs-on: [Windows, self-hosted]
     timeout-minutes: 180
-    continue-on-error: true
```

Comment:
Is the default false?
Can we just have it explicitly set to false?

Reply:
Yea, it's default to false.

Reply:
But it's doesn't matter now, because I set
```
    - name: result notification
      id: slack-notify
      if : always()
```

which will make sure this "slack-notify" keep running no matter CTS fail or not.